Slambook/index.js

```diff
@@ -25,8 +26,47 @@ let nickname= document.getElementsByName('nickname');
 let songForMe= document.getElementsByName('songForMe');
 let shareOpinion= document.getElementsByName('shareOpinion');
 
-console.log(contactName,relationship, likeInMe, hateInMe,deadReaction,firstImpression,beautifulMessage,nickname,songForMe,shareOpinion);
-// Write your code above this line
-}
+let inputNames= [contactName, relationship, likeInMe, hateInMe, deadReaction, firstImpression, beautifulMessage, nickname, songForMe, shareOpinion];
+
+let contactNameValue= document.querySelector(`input[name = 'contactName']`).value;
+let relationshipValue= document.querySelector(`[name = 'relationship']`).value;
+let likeInMeValue= document.querySelector(`[name = 'likeInMe']`).value;
+let hateInMeValue= document.querySelector(`[name= 'hateInMe']`).value;
+let deadReactionValue= document.querySelector(`[name= 'deadReaction']`).value;
+let firstImpressionValue= document.querySelector(`[name='firstImpression']`).value;
+let beautifulMessageValue= document.querySelector(`[name='beautifulMessage']`).value;
+let nicknameValue= document.querySelector(`[name='nickname']`).value;
+let songForMeValue= document.querySelector(`[name='songForMe']`).value; 
+let shareOpinionValue= document.querySelector(`[name='shareOpinion']`).value;
+
+let slamEntry= {
+    contactName: contactNameValue,

```

Comment:
The 'after' method is deprecated. Consider using the 'insertAdjacentElement' method with the 'after' position instead.